src/app/services/ccd-config/ccd-case.config.ts

```diff
@@ -64,6 +64,14 @@ export class AppConfig extends AbstractAppConfig {
     return this.config.remote_document_management_url;
   }
 
+  public getHrsUrl() {
```

Comment:
Could do with a return type. Also, what is Hrs?

Reply:
Absolutely no idea. This is here due to a breaking change that was introduced in the CCD Case UI Toolkit. Good call, though - I'll add a return type. Same for the one below. To be fair, it looks like they all need one! 🤦‍♂️

Reply:
Done.

Reply:
Great, thanks.